### ISSUE TITLE
Close Menu Item Shows up in the middle of the menu options

### ISSUE THREAD

**Issue by JLLeitschuh** [angular-ui/ui-grid#4969]
The menu should be at the bottom of the menu list.
This UI component only shows up when tabbing through the menu options.

This menu option should appear at the bottom of the menu.
![screen shot 2016-01-14 at 1 49 00 pm](https://cloud.githubusercontent.com/assets/1323708/12334068/f21e56a4-bac5-11e5-9a20-8b71c63f540a.png)

**Comment by jmalkan** [comment]
Can you provide a plunkr?

**Comment by JLLeitschuh** [comment]
Hahaha. This is quite an old issue. I'm now working at a completely new company.
I think this only showed up if you were tabbing through the menus. I think the `Close` button only showed up when the browser has focus on the menu item.

Talk to @mportuga to see if this is still an issue a Kronos.

The "Close" button might have just totally been removed too. I don't know.

**Comment by jmalkan** [comment]
yeah we are going through the issues to find which ones need still fixing.

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
